watchmal/dataset/cnn_mpmt/cnn_mpmt_dataset.py

```diff
@@ -42,9 +42,6 @@ def __init__(self, h5file, mpmt_positions_file, padding_type=None, transforms=No
         transforms: sequence of string
             List of random transforms to apply to data before passing to CNN for data augmentation. Each element of the
             list should be the name of a method of this class that performs the transformation
-        collapse_arrays: bool
-            Whether to collapse the image-like CNN arrays to a single channels containing the sum of other channels.
-            i.e. provide the sum of PMT charges in each mPMT instead of providing all PMT charges.
```

Comment:
Sorry for another request that I missed previously. Since the init function parameters have changed, the doc string (comment with documentation of the function) has not been fully updated. The documentation of the removed `collapse_arrays` parameter has been removed, but the new parameters `mode`, `collapse_mode`, `scaling_charge` and `scaling_time` need to have documentation added to the doc string. Please could you document these new parameters in place of the removed collapse_arrays documentation?

Reply:
Hi @nickwp, I added the new comments. Please, let me know if they are ok.

Reply:
Looks great, thanks